src/bacnet/basic/object/sc_netport.c

```diff
@@ -175,11 +186,22 @@ bool Network_Port_SC_Failover_Hub_URI_Set(uint32_t object_instance, char *str)
     BACNET_SC_PARAMS *params = Network_Port_SC_Params(object_instance);
     if (!params)
         return false;
-    snprintf(params->SC_Failover_Hub_URI, sizeof(params->SC_Failover_Hub_URI),
-        "%s", str);
-    
+    snprintf(params->SC_Failover_Hub_URI,
+        sizeof(params->SC_Failover_Hub_URI), "%s", str);
+
```

Comment:
@Michail-Antropov This difference shows that Network_Port_Changes_Pending_Set() was dropped from the definition of Network_Port_SC_Primary_Hub_URI_Set(). Is that intentional?

Reply:
Yes, the flag Changes Pending is set Network_Port_Write_Property(). Otherwise, the flag will be set for each setter call, incl. during normal initialization the SC_Primary_Hub_URI.